src/Valit/Result/IValitResult.cs

```diff
@@ -2,7 +2,7 @@ namespace Valit
 {
     public interface IValitResult
     {
-         bool Succeeded { get; }
+         bool Succeded { get; }
```

Comment:
Succeeded was the correct one probably

Reply:
You are right I guess... Most google searches go to Succeeded... 
I remember that I have stumbled across that one first: https://en.wiktionary.org/wiki/succeded